sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -364,6 +879,100 @@ public async Task RecognizeLinkedEntitiesBatchWithErrorTest()
             Assert.Throws<InvalidOperationException>(() => results[1].Entities.GetType());
```

Comment:
Should we assert the exception message, since it is a custom message very specific to the TA library atm?